src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -396,6 +412,256 @@ private bool SkipGeneratedBranchesForExceptionHandlers(MethodDefinition methodDe
             return _compilerGeneratedBranchesToExclude[methodDefinition.FullName].Contains(instruction.Offset);
         }
 
+        private bool SkipGeneratedBranchesForAwaitForeach(List<Instruction> instructions, Instruction instruction)
```

Comment:
Yep.  There are a couple of other existing **Skip...** helper methods that aren't static, so I'll go ahead and fix those, too.

Reply:
Thx

Reply:
Note: One of them (`SkipGeneratedBranchesForExceptionHandlers`) can't be made `static`, since it references the non-static field `_compilerGeneratedBranchesToExclude`.